modules/ROOT/pages/database-administration/standard-databases/errors.adoc

```diff
@@ -60,57 +60,24 @@ A database management operation may fail for a number of reasons.
 For example, if the file system instance has incorrect permissions, or Neo4j itself is misconfigured.
 As a result, the contents of the `statusMessage` column in the `SHOW DATABASE` query results may vary significantly.
 
-However, databases may only be in one of a select number of states:
+However, databases may only be in one of a select number of states (see xref::database-administration/standard-databases/manage-databases.adoc#database-states[Database states] for more information):
```

Comment:
Maybe we could add the link after the states' list so we don't interrupt the reading flow.

Reply:
Yes, I agree, that's better!